code/_globalvars/lists/faxes.dm

```diff
@@ -0,0 +1,21 @@
+/**
+ * This defines the list of faxes managed by the server administrators. They are not physically present in
+ * the game, but are shown in the fax list as existing.
+ * Lists:
+ * * additional_faxes_list - A list of "legal" faxes available with authorization.
+ * * syndicate_faxes_list - List of faxes available after hacking.
+ *
+ * The list consists of the following elements:
+ * * fax_name - The name displayed in the fax list.
+ * * button_color - The color of this fax button in the list of all faxes.
+ */
+GLOBAL_LIST_INIT(additional_faxes_list, list(
+	list("fax_name" = "Central Command", "button_color" = "#34c924"),
+	list("fax_name" = "Clown Planet", "button_color" = "#f4c800"),
+))
+
+GLOBAL_LIST_INIT(syndicate_faxes_list, list(
+	list("fax_name" = "Syndicate Coordination Center", "button_color" = "#ff0000"),
+	list("fax_name" = "Federation of Wizards", "button_color" = "#8b00ff"),
+	list("fax_name" = "Nar-Sie Church", "button_color" = "#8b0000"),
+))
```

Comment:
While this seems nice, I don't think I agree with it being in-game, unless you change the methods to gain these. 
My thinking is limiting individual faxes to specific accesses (Centcom is Cap/Lawyer, Clown planet is Clown, Syndicate is Syndie access (the one that Agent IDs have), Nar'Sie could be maybe a culted fax using twisted construction, wizard federation I can't really explain at all).

Reply:
I understand your criticism. But I still want to defend my approach. First of all, I'm afraid that organizing access for each role would make the system very complicated. Secondly, giving access to certain facsimiles for certain roles will narrow the framework for role-playing. I see no reason to limit the captain to be able to write complaints to Planet of the Clowns, and the clown to write unfunny jokes for the CC. Third, I created the list specifically based on requests from developers of other builds based on TG. They would like to be able to designate their own organizations and factions for their servers. For TG, I entered such organizations for example, and nothing stops them from tweaking or deleting.